index.html

```diff
@@ -61,7 +61,7 @@
 
   <section id="getting-started">
     <aside>
-    <h3>Alright. How do I get started?</h3>
```

Comment:
How about `OK, how do I get started`? I think "All right" means that all of something is correct, which is not useful here.